libs/community/langchain_community/document_loaders/glue_catalog.py

```diff
@@ -0,0 +1,103 @@
+from typing import Any, Dict, Iterator, List, Optional
+
+from langchain_core.documents import Document
+
+from langchain_community.document_loaders.base import BaseLoader
+
+
+class GlueCatalogLoader(BaseLoader):
+    """Load table schemas from AWS Glue.
+
+    This loader fetches the schema of each table within a specified AWS Glue database.
+    The schema details include column names and their data types, similar to pandas dtype representation.
+
+    AWS credentials are automatically loaded using boto3, following the standard AWS method:
+    https://boto3.amazonaws.com/v1/documentation/api/latest/guide/credentials.html
+
+    If a specific AWS profile is required, it can be specified and will be used to establish the session.
+    """
+
+    def __init__(
+        self,
+        database: str,
+        profile_name: Optional[str] = None,
+        table_filter: Optional[List[str]] = None,
+    ):
+        """Initialize Glue database loader.
+
+        Args:
+            database: The name of the Glue database from which to load table schemas.
+            profile_name: Optional. The name of the AWS profile to use for credentials.
+            table_filter: Optional. The list of the table to to filter.
```

Comment:
What does it mean to have a list of tables to filter? Maybe paraphrase the doc-string?

```suggestion
            table_filter: Optional. The list of the table to filter.
```

Reply:
You are right, it was not clear. I rewrited this part to be more explicit.